bin/setup

```diff
@@ -3,6 +3,5 @@ set -euo pipefail
 IFS=$'\n\t'
 set -vx
 
-bundle install
+bundle install --path vendor/bundle
```

Comment:
Just curious about whats the advantage of `--path vendor/bundle`?

Reply:
gonna remove this and fix the problem I needed it for on my machine